generate/generate.go

```diff
@@ -1062,6 +1074,23 @@ func mapType(t string) string {
 	}
 }
 
+func mapResponseType(t string) string {
+        switch t {
+        case "boolean":
+                return "bool"
+        case "short", "int", "integer", "long":
+                return "int64"
```

Comment:
Wondering if a better solution would be to make a separate case for `long` so we return an int64 only for that one, but in all locations?